debian/control

```diff
@@ -26,7 +26,7 @@ Depends: python (>= 2.7.1),
          python-serial,
          python-netifaces,
 # PIL is only needed in test cases, and indirectly by scipy for some image processing
-         python-imaging,
```

Comment:
It was called python-pil in 16.04 already, right? Why isn't python-pil in the list?

Reply:
This was to support 12.04, which we have supported until Odemis 2.9 (ie, the latest stable version) :-) 
Now that the next release will not need to support 12.04, we can finally change it. We could have different control files for each Ubuntu release that we support, but unfortunately with the standard Ubuntu (and Debian) build system, it's a pain.